db/mongo/client.go

```diff
@@ -32,7 +32,7 @@ func NewClient(conf db.Option) (db.DataStore, error) {
 	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
 	cli, err := mongo.Connect(ctx, options.Client().ApplyURI("mongodb://localhost:27017"))
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "database connection failed")
```

Comment:
Lets use proper messaging. i-e "unable to connect to db"

Reply:
ok